src/Enhavo/Bundle/AppBundle/Command/GenerateRoutingCommand.php

```diff
@@ -31,6 +31,11 @@ protected function configure()
                 InputArgument::REQUIRED,
                 'What is the name of the resource?'
             )
+            ->addArgument(
```

Comment:
I think its better to use options than arguments. And order should be the default value

```
enhavo:routing:generate --sorting myApp myResource
```

```
enhavo:routing:generate --sorting="myOrderProperty" myApp myResource
```